e2e/src/base/BasePage.ts

```diff
@@ -0,0 +1,23 @@
+import { Page } from '@playwright/test';
+
+export default abstract class BasePage {
+    protected readonly BASE_PAGE = 'https://vadimnastoyashchy.github.io/';
```

Comment:
Better to store base url in playwright config file

Reply:
Changed